interpreter/test/globals.wast

```diff
@@ -95,3 +95,15 @@
   (module (global i32 (get_global 1)) (global i32 (i32.const 0)))
   "unknown global"
 )
+
+;;(module
```

Comment:
No, I think it's good to have. But in this case, why not uncomment it? Can't harm.